### PR TITLE
Inconsistent plans

### PR DESCRIPTION
**Pull request by moshthepitt**
This PR attempts to ensure that the values used for plans are consistent with the ones used in the PlanForm component, which is based on the Reveal data dictionary: https://docs.google.com/spreadsheets/d/1kGPgyF0eJdag19FFL1Oog2gdphdPtKobY4EQ0gMgtgw/edit#gid=1805975071

### CODE REVIEW

**Comment by moshthepitt** [comment]
There are some huge snapshots touched by this PR, but it is a bit urgent so I have not worked on cleaning the snapshot tests.  There are issues to do that.

**Comment by moshthepitt** [comment]
Added a missing issue for cleanup: https://github.com/onaio/reveal-frontend/issues/552

**Comment by moshthepitt** [comment]
Related Jira issue: https://smartregister.atlassian.net/browse/RVL-610

File: src/constants.tsx [hunk 1]
```diff
@@ -99,10 +100,8 @@ export const STRUCTURES = 'structure(s)';
 export type STRUCTURES = typeof STRUCTURES;
 export const PERSONS = 'person(s)';
 export type PERSONS = typeof PERSONS;
-export const CASE_TRIGGERED = 'Case-triggered';
-export type CASE_TRIGGERED = typeof CASE_TRIGGERED;
-export const ROUTINE = 'Routine';
-export type ROUTINE = typeof ROUTINE;
+export const CASE_TRIGGERED: FIReasonType = FIReasons[1];
+export const ROUTINE: FIReasonType = FIReasons[0];
```
Comment (cKellyDesign) [inline src/constants.tsx]: Nice 👍

**Pull request merged**